packages/cli/lib/chapter.js

```diff
@@ -234,13 +234,14 @@ class Chapter {
 
     function reformatImgSources() {
       chapter.images.attr("src", (index, src) => {
+        const imgPath = src.replace(/^(\.+\/)+img\//, "");
         let localPath = path
           .resolve(
             path.join(chapter.config.outputDir, chapter.config.imageDir),
-            src
+            imgPath
           )
           .replace(/\/localhost:1313/, chapter.config.outputDir);
-        return "./" + localPath;
+        return localPath;
```

Comment:
the image `src` that's from the template here uses the `relURL` filter, which adds the appropriate relative prefix to the image path - but this is relative to the directory structure of `site`, so here when we attempt to get the local path to the image, the relative path is incorrect for images in template nested more than one level deep. This change adds a regex to remove the relative part of the path (../.., ../) up to `img`, and then uses it to resolve the local path to the image.